app/views/subscriptions/_subscription_fields.html.erb

```diff
@@ -13,7 +13,11 @@
   </div>
   <div class="row">
     <p class="form-description">Enter a Concept ID for a single collection. (Example: C1234567-PODAAC)</p>
-    <%= text_field_tag('subscription[CollectionConceptId]', @subscription['CollectionConceptId'], class: 'required full-width') %>
+    <% if edit %>
+      <%= text_field_tag('subscription[CollectionConceptId]', @subscription['CollectionConceptId'], class: 'required full-width disabled', disabled: 'disabled') %>
+    <% else %>
+      <%= text_field_tag('subscription[CollectionConceptId]', @subscription['CollectionConceptId'], class: 'required full-width') %>
+    <% end %>
   </div>
```

Comment:
I'm also wondering if we should have a comment as we do on the group form about certain fields not being editable

Reply:
is there any other forms documentation, such as test that checks for this? If so, then we can just move on, otherwise I am more in favor of documentation than not.

Reply:
the tests were updated to expect the fields to be disabled.

Reply:
and it was documented in the controller